src/gui/CMakeLists.txt

```diff
@@ -1489,6 +1491,9 @@ set(QGIS_GUI_UI_HDRS
   ${CMAKE_CURRENT_BINARY_DIR}/../ui/ui_qgstemporalcontrollerwidgetbase.h
   ${CMAKE_CURRENT_BINARY_DIR}/../ui/ui_qgsprocessingtinmeshdatawidgetbase.h
   ${CMAKE_CURRENT_BINARY_DIR}/../ui/ui_qgsexpressionpreviewbase.h
+  ${CMAKE_CURRENT_BINARY_DIR}/../ui/ui_qgsprovidersublayersdialogbase.h
+  ${CMAKE_CURRENT_BINARY_DIR}/../ui/ui_qgsdatumtransformdialogbase.h
+  ${CMAKE_CURRENT_BINARY_DIR}/../ui/ui_qgscoordinateoperationwidgetbase.h
```

Comment:
Unrelated?

Reply:
Semi-related - included headers refer to these headers. Though might indeed also come from another include path from qgsapplayerhandling.cpp which I was porting to KADAS. I can split it out to a separate commit if preferred.

Reply:
No need to, just wanted to be sure it's not an accident.